commit 9eb995ef8a34d552077b2af02787574a6534b1e2
Author: Tasos Latsas <tlatsas2000@gmail.com>
Date:   Sat Jun 30 11:07:45 2012 +0300

    wifi-menu: fix name of wireless key variable
    
    Dialog returns the wireless key in variable $key. Uppercase
    'KEY' is used later to create the profile.
    
    Signed-off-by: Tasos Latsas <tlatsas2000@gmail.com>

diff --git a/scripts/wifi-menu b/scripts/wifi-menu
index e182f25..ba8eb47 100755
--- a/scripts/wifi-menu
+++ b/scripts/wifi-menu
@@ -115,7 +115,7 @@ create_profile()
         [[ "$OBSCURE" ]] && box="--insecure --passwordbox" || box="--inputbox"
         msg="Enter $security security key for\n'$1'"
         key=$(dialog $box "$msg" 10 40 --stdout) || return $?
-        [[ "${#KEY}" -ge 8 && "${#KEY}" -le 63 ]] || return 4
+        [[ "${#key}" -ge 8 && "${#key}" -le 63 ]] || return 4
         if [[ "$OBSCURE" ]]; then
             key=$(wpa_passphrase "$1" "$key" | grep -m 1 "^[[:space:]]*psk=")
             key="KEY=${key#*psk=}"
